src/main/java/seedu/address/logic/commands/AddModuleCommand.java

```diff
@@ -18,7 +18,7 @@
     public static final String MESSAGE_USAGE = AddCommand.COMMAND_WORD + " " + COMMAND_TYPE
             + ": Adds a module to TAB. "
             + "Parameters: "
-            + PREFIX_MODULE_NAME + "MODULE NAME "
+            + PREFIX_MODULE_NAME + "MODULE NAME \n"
```

Comment:
Should we create an issue to standardize on how the `MESSAGE_USAGE` should look for all of the commands?

i.e. using line breaks so as to prevent the message from appearing too long in the Command Result box

Reply:
Yes I think we should

Reply:
yeah agree